spaces/domains.json

```
@@ -440,5 +440,6 @@
   "vote.ewd.green": "carbonpaws.eth",
   "vote.litedex.io": "litedex.eth",
   "vote.matcha.fi": "streamingfast.eth",
-  "vote.bsc-mvb.io": "mvb.eth"
+  "vote.bsc-mvb.io": "mvb.eth",
+  "vote.liquidrium.com": "liquidrium.eth",
```

Please delete the comma at the end of the line.
```suggestion
  "vote.liquidrium.com": "liquidrium.eth"
```